changelog/_unreleased/2020-09-17-allow-url-query-parameters-in-search-api-requests.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Allow URL query parameters in POST /api/v{number}/search/{entity} requests
+issue: NEXT-7952
+author: Carsten Dietrich
+author_email: 3203968+carstendietrich@users.noreply.github.com
+author_github: @carstendietrich
+---
+# Core
+* Update `\Shopware\Core\Framework\DataAbstractionLayer\Search\RequestCriteriaBuilder::handleRequest`
```

Comment:
the changelog does not comply with our new conventions ☺️

Reply:
Ah my bad, I was in a hurry to finish the changelog before the final presentation. Updated the changelog accordingly :ok_hand: